rllib/algorithms/ppo/tests/test_ppo_with_rl_module.py

```diff
@@ -176,6 +251,134 @@ def test_ppo_exploration_setup(self):
             check(np.mean(actions), 1.5, atol=0.2)
             trainer.stop()
 
+    def test_rollouts(self):
```

Comment:
Took this structure from two test methods that @kouroshHakha discarded and extended it to include LSTM stuff.